packages/sdk/src/utils/validators.ts

```diff
@@ -97,7 +97,8 @@ export const grantedAccessSchema = () =>
     .noUnknown()
     .default(undefined);
 
-export const urlArraySchema = () => array().of(urlSchema());
+export const urlArraySchema = () =>
+  array().of(string().matches(/^http[s]?:\/\//, '${value} should be a url'));
```

Comment:
Specific array validator and use `${value}` here instead of `${path}`.
Otherwise we get something like "[0] is not a string" which is of limited help.